editor/src/app/profile-dropdown/profile-dropdown.component.ts

```diff
@@ -44,10 +55,21 @@ import { Router } from '@angular/router';
     }
   `]
 })
-export class ProfileDropdownComponent {
+export class ProfileDropdownComponent implements OnInit {
+
+  user: Observable<UserStateModel>;
+
   constructor(
     private appStateService: AppStateService,
-    private router: Router) {
+    private router: Router,
+    private store: Store) {
+  }
+
+  ngOnInit() {
+
+    this.store.select(UserState).subscribe((userState) => {
```

Comment:
When creating an infinite subscription on component creation, it should be destroyed on component destruction.
`this.sub = this.store.select(UserState).subscribe(...)`
```ts
ngOnDestroy() {
    this.sub.unsubscribe();
}
```
https://brianflove.com/2016/12/11/anguar-2-unsubscribe-observables/